<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_y">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="ChartProgress">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>381</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtn_render">
       <property name="text">
        <string>渲染图表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_SelectData">
       <property name="text">
        <string>保存数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_OpenData">
       <property name="text">
        <string>打开数据</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_Type">
       <item>
        <property name="text">
         <string>饼状图/Pie</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>柱状图/Bar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>折线图/Line</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>更多正在开发中</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_LangSelect">
       <item>
        <property name="text">
         <string>简体中文</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>English</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_oklang">
       <property name="text">
        <string>确认语言</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="ledit_x">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ledit_y">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_ok_x">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_ok_y">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>41</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_ver">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>220</y>
      <width>581</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>软件版本：1.0.0                 制作者：MojaveHao</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_title">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>图表生成器   ChartWorking</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
